src/Raven.Studio/typescript/components/pages/database/indexes/errors/IndexErrors.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Col, Row } from "reactstrap";
+import { AboutViewHeading } from "components/common/AboutView";
+import IndexErrorsActions from "components/pages/database/indexes/errors/IndexErrorsActions";
+import { IndexErrorsPanel } from "components/pages/database/indexes/errors/IndexErrorsPanel";
+import { useIndexErrors } from "components/pages/database/indexes/errors/hooks/useIndexErrors";
+import useIndexErrorsTable from "components/pages/database/indexes/errors/hooks/useIndexErrorsTable";
+import IndexErrorsAboutView from "components/pages/database/indexes/errors/IndexErrorsAboutView";
+
+export default function IndexErrors() {
+    const { indexErrorsPanelTable } = useIndexErrorsTable();
+    const { errorInfoItems, asyncFetchAllErrorCount, erroredIndexNames, erroredActionNames } = useIndexErrors();
+
+    return (
+        <div className="content-margin">
+            <Col xxl={12}>
+                <Row className="gy-sm">
+                    <Col>
+                        <div className="flex-shrink-0 hstack gap-2 align-items-start">
+                            <AboutViewHeading icon="index-errors" title="Index Errors" />
+                        </div>
+                        <IndexErrorsActions
+                            refresh={asyncFetchAllErrorCount.execute}
+                            isLoading={asyncFetchAllErrorCount.loading}
+                            erroredActionNames={erroredActionNames}
+                            erroredIndexNames={erroredIndexNames}
+                            filters={indexErrorsPanelTable.getState().columnFilters}
+                            setFilters={indexErrorsPanelTable.setColumnFilters}
+                        />
+                        {errorInfoItems.map((errorItem, index) => (
+                            <IndexErrorsPanel
+                                key={index}
+                                asyncFetchAllErrorCount={asyncFetchAllErrorCount}
+                                errorItem={errorItem}
+                                table={indexErrorsPanelTable}
```

Comment:
What if you open multiple panels? The content of the table will not be replaced?